iqrm/core.py

```diff
@@ -93,6 +96,14 @@ def iqrm_mask(x, radius=5, threshold=3.0):
             votes_cast[j].add(i)
```

Comment:
Can I ask you a favour and add the following lines just after `for i, j in zip(I, J):` 
```
i = int(i)
j = int(j)
```

I noticed that otherwise the votes dictionaries contain a mix of Python `int` and `np.int64` and I think that might cause issues. It'd definitely my bad though :smile: